src/System.Private.Uri/src/Resources/Strings.resx

```diff
@@ -258,4 +258,13 @@
   <data name="IO_PathTooLong" xml:space="preserve">
     <value>The specified file name or path is too long, or a component of the specified path is too long.</value>
   </data>
+  <data name="Argument_ExtraNotValid" xml:space="preserve">
+    <value>Extra portion of URI not valid</value>
+  </data>
+  <data name="Argument_InvalidUriPart" xml:space="preserve">
+    <value>Uri part is not is not valid.</value>
```

Comment:
"Uri part is not is not valid" doesn't make sense and has repeated words.

Can you explain more what part of the code would use this error string?  This will help figure out the right text.